image/store_test.go

```diff
@@ -6,15 +6,15 @@ import (
 
 	"github.com/docker/docker/layer"
 	"gotest.tools/v3/assert"
-	"gotest.tools/v3/assert/cmp"
+	is "gotest.tools/v3/assert/cmp"
 )
 
 func TestCreate(t *testing.T) {
-	is, cleanup := defaultImageStore(t)
+	imgStore, cleanup := defaultImageStore(t)
```

Comment:
TBH; slightly on the fence for this file, as it's causing a larger diff than I'd hoped due to many occurrences of `is` as name for "imagestore"